src/main/java/gui/MonitorScene.java

```diff
@@ -52,22 +52,17 @@ public MonitorScene(final Controller crtl) {
     /**
      * monitorScene.
      * Creates the scene where host can monitor the game.
-     * @param width of the scene
-     * @param height of the scene
      * @param primaryStage starting stage
      * @param stylesheet current stylesheet
      * @return monitorScene
      */
-    public Scene createMonitorScene(final int width,
-                                    final int height,
-                                    final Stage primaryStage,
-                                    final String stylesheet) {
+    public Scene createMonitorScene(final Stage primaryStage, final String stylesheet) {
```

Comment:
Does this have anything to do with this pull request?

Reply:
yes [here](https://github.com/egedebruin/Trackscape/pull/30/commits/2de418418ba3f9caad7b68511b1cd4d1fde0552f) you can see why I had to make this commit.

Reply:
Kort samengevat, de merge kan niet gebeuren zonder dat BCH slaagt. BCH slaagde niet omdat   een methode 5 input parameters had. width en height waren echter onnodig dus die heb ik weggehaald. Hierdoor heeft die methode nog maar 3 parameters, wat wel geaccepteerd word. Het verwijderen van width en height in die methode had tot gevolg dat andere methodes die ook width en height hadden staan ook aangepast moesten worden.